content/en/blog/2019/introducing-istio-operator/index.md

```diff
@@ -0,0 +1,161 @@
+CRD (Custom Resource Definition) which is validated.
+
+The newly released Istio operator simplifies the common administrative tasks of installation,
+upgrade, and complex configuration changes for Istio.
+Starting with Istio 1.4, the [Helm installation](/docs/setup/install/helm/) approach has been deprecated
+in favor of a new operator-based [installation using {{< istioctl >}}](/docs/setup/install/operator/).
+Upgrading from Istio 1.4 onward (that is, versions not initially installed with Helm)
+will also be done using a new [{{< istioctl >}} upgrade feature](/docs/setup/upgrade/istioctl-upgrade/).
+
+## Istio Control Plane API
+
+An operator implementation requires a CRD (Custom Resource Definition) to define the API.
+The Istio operator CRD is defined by the [`IstioControlPlane` API](/docs/reference/config/istio.operator.v1alpha12.pb/).
+This API supports all of Istio's current [configuration profiles](/docs/setup/additional-setup/config-profiles/)
+using a single field to select the profile. For example, the following CRD configures Istio using the `demo` profile:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+metadata:
+  namespace: istio-operator
+  name: example-istiocontrolplane
+spec:
+  profile: demo
+{{< /text >}}
+
+You can then customize the configuration with additional settings. For example, to disable telemetry:
+
+{{< text yaml >}}
+apiVersion: install.istio.io/v1alpha2
+kind: IstioControlPlane
+metadata:
+  namespace: istio-operator
+  name: example-istiocontrolplane
+spec:
+  profile: demo
+  telemetry:
+    enabled: false
+{{< /text >}}
+
+## Istio Controller (alpha)
+
+An operator implementation also requires a Kubernetes controller to continuously monitors the CRD
+and apply the corresponding configuration changes.
+The Istio controller is a pod in your cluster that reacts to changes in an
+`IstioControlPlane` CRD for the cluster by updating the Istio installation configuration.
+
+In the 1.4 release, the Istio controller is still in the alpha phase of development and not fully
+integrated with `istioctl`. It is, however, available for experimentation using `kubectl` commands.
+For example, to install the controller and a default version of Istio into your cluster,
+run the following command:
+
+{{< text bash >}}
+$ kubectl apply -f https://<repo URL>/<version>/operator-profile-default.yaml
+{{< /text >}}
+
+You can then make changes to the Istio installation configuration:
+
+{{< text bash >}}
+$ kubectl edit istiocontrolplane example-istiocontrolplane -n istio-system
+{{< /text >}}
+
+To upgrade to a new version of Istio, run:
+
+{{< text bash >}}
+$ kubectl apply -f https://<repo URL>/<new version>/operator-profile-default.yaml
+{{< /text >}}
+
+Refer to [standalone operator](/docs/ops/setup/standalone-operator/) for more details.
+
+## Using the {{< istioctl >}} CLI
+
+In the meantime, until the controller is thouroughly tested and better integrated with `istioctl`,
```

Comment:
Actually I think controller is not likely to become the preferred method of install and admin for security reasons. I think it's better to steer users to istioctl first in the blog.